Added variants property

wesleimp opened the pull request.

wesleimp committed:
Added variants property

wesleimp committed:
Merge pull request #1 from wesleimp/added-variants

Added variants property

silviogreuel committed:
Typo and using IEnumerable instead of ICollection

silviogreuel committed:
Variants name should be in singular

silviogreuel committed:
Adding project to test variants

silviogreuel committed:
Adding docs

silviogreuel committed:
Listing variants by toggle

ivarconr commented:
Nice! :fire:

silviogreuel committed:
Adding weighted variant result

silviogreuel committed:
Adding payload to variant

silviogreuel committed:
Removing unused usings

silviogreuel committed:
Typo in summary and adding extra information

silviogreuel commented on src/Unleash/DefaultUnleash.cs:
```diff
@@ -88,6 +91,71 @@ private bool CheckIsEnabled(string toggleName, UnleashContext context, bool defa
+            return null;
+        }
+
+        private int? GetWeightedPosition(int[] weights)
```
In need of a better solution here, or this is ok?

silviogreuel committed:
Adding docker-compose and Dockerfile to support some sample usage

silviogreuel committed:
Changing appveyor.yml to build on a Visual Studio 2019 os

silviogreuel committed:
Avoiding installing -pre releases on appveyor

silviogreuel committed:
Fixing on appveyor gitversion tool to be on version 4.0.0

Solution from: https://help.appveyor.com/discussions/problems/24758-gitversion-5-not-working-on-visual-studio-2019-build

silviogreuel commented:
@stiano and @ivarconr, maybe, turn this repository as official sdk under unleash domain? to be more central and to gain some velocity on some replies/isues? tnks lads, great work you both...

silviogreuel committed:
Ignoring AssemblyInfo on .net-core project (ci)

Based on this solution: https://stackoverflow.com/a/55573656

silviogreuel commented:
Hey @stiano @ivarconr, Just to notify, using this repository as base for https://github.com/Unleash/unleash-client-core
